### PR TITLE
test: ruby custom gemfile test

### PR DESCRIPTION
**Pull request by lili2311**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?

Include a test that shows targeting a custom Gemfile with --file will not be able to determine the lockfile name and asks the user to use the lockfile.

### CODE REVIEW

File: test/acceptance/cli-test/cli-test.ruby.spec.ts [hunk 1]
```diff
@@ -78,6 +78,24 @@ export const RubyTests: AcceptanceTests = {
...
+      utils.chdirWorkspaces();
+      try {
+        await params.cli.test('ruby-app-custom-names', {
+          file: '123.gemfile',
+          packageManager: 'rubygems',
+        });
+      } catch (e) {
+        t.match(
+          e.message,
+          'if this is a custom file name re-run with --file=path/to/custom.gemfile.lock --package-manager=rubygems',
```
Comment (anthogez) [inline test/acceptance/cli-test/cli-test.ruby.spec.ts:94]: Shouldn’t the message say `if this is a custom file name re-run with --file=path/to/anything-food.gemfile --package-manager=rubygems`

without `.lock`?

Comment (lili2311) [reply test/acceptance/cli-test/cli-test.ruby.spec.ts:94]: the message is correct, we require the `.lock` file only for a scan. We can auto detect it from a Gemfile normally if it follows a standard pattern, but id the pattern is custom we can't guess where `.lock` might be. We do by default try to append it to the end of the Gemfile name, let me check this again as it looks like in this case it should have worked & I need to change the example to a complex one where .lock is not at the end

Comment (lili2311) [reply test/acceptance/cli-test/cli-test.ruby.spec.ts:130]: Okay updated!

Comment (anthogez) [reply test/acceptance/cli-test/cli-test.ruby.spec.ts:130]: Thanks! 🎉

**Review by anthogez — APPROVED** [review]

**Pull request merged**